src/Security/Authentication/OAuth/src/OAuthHandler.cs

```diff
@@ -177,7 +178,7 @@ protected virtual async Task<OAuthTokenResponse> ExchangeCodeAsync(string code,
             var response = await Backchannel.SendAsync(requestMessage, Context.RequestAborted);
             if (response.IsSuccessStatusCode)
             {
-                var payload = JObject.Parse(await response.Content.ReadAsStringAsync());
+                var payload = JsonDocument.Parse(await response.Content.ReadAsStringAsync());
```

Comment:
Who disposes this?

Reply:
Oh, `OAuthTokenResponse` is now disposable. I see. So the caller must remember to dispose it. K :)